### PR TITLE
Add the option to delete all the comments.

### PR DESCRIPTION
**Pull request by ancamaria99**

### CODE REVIEW

File: ancar/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -41,20 +41,33 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
 
-      Gson gson = new Gson();
-
       String valueParameter = request.getParameter("value");
       int numberOfComments = Integer.parseInt(valueParameter);
 
-      List<String> commentsToPrint = comments.subList(0, numberOfComments);
+      Gson gson = new Gson();
+
+      if (comments.size() == 0) {
```
Comment (lanseg) [inline ancar/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:56]:
```java 
comments.isEmpty();
```

would be a bit easier to read

File: ancar/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -41,20 +41,33 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
+
+    //If the user requests more comments than there are available, print all.
+      if (comments.size() < numberOfComments) {
+          response.setContentType("application/json");
+          response.getWriter().println(gson.toJson(comments));
+          return ;
+      }
 
-      response.setContentType("application/json;");
+      commentsToPrint = comments.subList(0, numberOfComments);
```
Comment (lanseg) [inline ancar/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:69]: Variable is used only here, makes sense to declare it here

File: ancar/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 3]
```diff
@@ -41,20 +41,33 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
...
 
       for (Entity entity : results.asIterable()) {
         String comment = (String) entity.getProperty("comment");
         comments.add(comment);
       }
 
-      Gson gson = new Gson();
-
       String valueParameter = request.getParameter("value");
       int numberOfComments = Integer.parseInt(valueParameter);
```
Comment (lanseg) [inline ancar/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:52]: No need to declare a variable here,
```java
int numberOfComments = Integer.parseInt(request.getParameter("value"));
``` 
would be also fine

Comment (ancamaria99) [reply ancar/portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:52]: should I resolve all of your suggestions and create a new pull request?

**Pull request merged**